apps/web/src/components/WidgetIconResolver/WidgetIconResolver.jsx

```diff
@@ -2,12 +2,12 @@ import React from 'react'
 import Avatar from 'components/Avatar'
 import { Grid3x3 } from 'lucide-react'
 import Icon from 'components/Icon'
-import { widgetTypeResolver } from 'util/contextWidgets'
+import { widgetTypeInferrer } from '@hylo/shared/src/ContextWidgetPresenter'
```

Comment:
I would like us, for a couple of reasons for now to keep the context for these Presenters. So in this and later cases `import { ContextWidgetPresenter } from '@hylo/shared'` and then use `ContextWidgetPresenter.widgetTypeInferrer()` in the component. I simply find that easier to grok when reading a component, but I also want to be able to easily move these references around once I complete sorting out the shared stuff soon, in which case I do think these presenters will go into a front-ends only shared package where the rest of the graphql and URQL config will also go.

Reply:
This is also my general preference for things coming from @hylo/shared. To restructure the top level module and then use it as context when calling the method in the component.

Reply:
Sorry, I was a little off there. I can see why you did it this way... Leave it be here for now.